crates/rolldown/src/runtime/runtime-extra-dev.js

```diff
@@ -125,6 +125,8 @@ class DevRuntime {
   createEsmInitializer = (fn, res) => () => (fn && (res = fn(fn = 0)), res)
   // __commonJSMin
   createCjsInitializer = (cb, mod) => () => (mod || cb((mod = { exports: {} }).exports, mod), mod.exports)
+  // @ts-expect-error it is exits
+  __toESM = __toESM
```

Comment:
Here just using the simply way to import runtime symbol, copy it need to copy it's dependent symbols.